parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -425,13 +425,17 @@ try
     Update-DefenderPreferences
 
     if ($windowsSecureTlsEnabled) {
-        Write-Host "Enable secure TLS protocols"
-        try {
-            . C:\k\windowssecuretls.ps1
-            Enable-SecureTls
-        }
-        catch {
-            Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_ENABLE_SECURE_TLS -ErrorMessage $_
+        if ((Get-WindowsVersion) -eq "ltsc2022") {

```

Comment:
It may be better to use `-ne "1809"` so we can remove these code when 1809 is deprecated by searching "1809"